pkg/vmimage/data/etc/origin/node/bootstrap-node-config.yaml

```diff
@@ -20,22 +20,26 @@ imageConfig:
   latest: false
 iptablesSyncPeriod: "30s"
 kubeletArguments:
-  pod-manifest-path:
-  - /etc/origin/node/pods
   bootstrap-kubeconfig:
   - /etc/origin/node/bootstrap.kubeconfig
-  feature-gates:
-  - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
-  rotate-certificates:
-  - "true"
   cert-dir:
   - /etc/origin/node/certificates
   cloud-config:
   - /etc/origin/cloudprovider/azure.conf
   cloud-provider:
   - azure
   enable-controller-attach-detach:
-  - 'true'
+  - "true"
+  feature-gates:
+  - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+  max-pods:
+  - "50"
+  pod-manifest-path:
+  - /etc/origin/node/pods
+  rotate-certificates:
+  - "true"
+  runtime-request-timeout:
+  - 10m
```

Comment:
this is new, what does it do?

Reply:
How long kubelet will try to GET,PUT, to `docker` to get stuff out. Does not include watchers (like logs).